lib/inertia_rails/controller.rb

```diff
@@ -38,6 +38,11 @@ def redirect_back(fallback_location:, allow_other_host: true, **options)
 
     private
 
+    def inertia_layout
+      layout = ::InertiaRails.layout
+      layout.nil? ? true : layout
```

Comment:
What does returning `true` do here?

Reply:
I'll add an inline doc, it lets Rails know that the view should use a layout.